.rubocop.yml

```diff
@@ -45,3 +45,9 @@ Style/SingleSpaceBeforeFirstArg:
 
 Style/PercentLiteralDelimiters:
   Enabled: false
+
+Style/AlignParameters:
```

Comment:
Rubocop styles should be discussed and added to the main Rubocop guide so we don't have different best practices within the Team. https://github.com/standout/Coding/tree/master/ruby